app/src/main/java/com/endava/myendava/adapters/UsersAdapter.java

```diff
@@ -54,17 +56,16 @@ public void setData(List<User> users) {
 
     class UserViewHolder extends RecyclerView.ViewHolder {
 
-        View itemView;
+        @BindView(R.id.user_photo_image_view)
         CircleImageView photoImageView;
```

Comment:
no need to specify the component on all namings.. here is pretty clear that the photo is an ImageView while the name and grade is a TextView

Reply:
i,ve changed it to userImage, userName and userGrade